resources/views/layouts/app.blade.php

```diff
@@ -141,6 +141,9 @@ className: 'help',
                                     <li>
                                         <a href="{{ route('rehearsals.listAttendances') }}">{{ trans('nav.attendance_last_rehearsal') }}</a>
                                     </li>
+                                    <li>
+                                        <a href="{{ route('rehearsals.listAttendances', ['id' => 'all']) }}">{{ trans('nav.attendance_rehearsals') }}</a>
+                                    </li>
```

Comment:
I am not sure if this is intuitive for choir admins. However, I am willing to leave it as is, let them test it and make changes if necessary.